python/pylibvw.cc

```diff
@@ -59,7 +59,7 @@ vw_ptr my_initialize(std::string args)
 { if (args.find_first_of("--no_stdin") == std::string::npos)
     args += " --no_stdin";
   vw*foo = VW::initialize(args);
-  return boost::shared_ptr<vw>(foo, dont_delete_me);
```

Comment:
should we leave a comment in case things go downhill in the future?

Reply:
particularly bc we're not sure of the impact and we also had some memory leak reported so there's something quirky going on with the python bindings. maybe a comment would be helpful as a breadcrumb in a future investigation

Reply:
https://github.com/VowpalWabbit/vowpal_wabbit/issues/2516

Reply:
I think leaving the rationale here would be ok. The code makes sense, and if a problem comes up, the commit history will eventually lead back to this PR. Sound reasonable?